lib/types/src/schema/pa11y.ts

```diff
@@ -0,0 +1,8 @@
+import { SchemaOutput } from '../schema'
+
+export const Pa11ySchema = {
```

Comment:
**suggestion:** it would be good to support other [options](https://github.com/pa11y/pa11y#configuration) such as `wait`, or at least the options supported in https://github.com/Financial-Times/n-pa11y-config#configuration